awx/api/serializers.py

```diff
@@ -2828,25 +2829,27 @@ def to_representation(self, obj):
         return ret
 
     def validate(self, attrs):
-        if 'project' in self.fields and 'playbook' in self.fields:
+        if 'project' in self.fields and 'playbook' in self.fields and 'fqcn_role' in self.fields:
```

Comment:
I feel like a lot of these conditionals should be reworked, but for a first stab at it... these work.